modules/wsl-distro.nix

```
@@ -50,10 +50,16 @@ with builtins; with lib;
         };
       };
 
-      # WSL is closer to a container than anything else
-      boot.isContainer = true;
+      # We don't need a boot loader
+      boot.loader.grub.enable = false;
+      system.build.installBootLoader = pkgs.writeShellScript "fake-bootloader" "";
```

```suggestion
      system.build.installBootLoader = "${pkgs.coreutils}/bin/true"
```
isContainer solved it like this which is a bit shorter and one derivation less